components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/client/authentication/BasicAuthClientAuthenticatorTest.java

```diff
@@ -285,6 +286,14 @@ public void testGetClientIdErrorScenario(String headerName, String headerValue,
                 OAuthClientAuthnContext());
     }
 
+    @Test
+    public void testGetSupportedClientAuthenticationMethods() {
+
+        List<String> supportedAuthMethods = basicAuthClientAuthenticator.getSupportedClientAuthenticationMethods();
```

Comment:
Shall we asset the list size is also 2 to make sure additional things are not added.

Reply:
Fixed in 428fe44b8129a99fb54bc22701c96ca5c6dc0bff